analyzers/src/SonarAnalyzer.CSharp/SymbolicExecution/Roslyn/SecureRandomSeedsShouldNotBePredictable.cs

```diff
@@ -128,13 +146,15 @@ private ProgramState ProcessStringToBytes(ProgramState state, IInvocationOperati
             && state[value]?.HasConstraint(CryptographicSeedConstraint.Predictable) is true;
     }
 
-    private static ProgramState ProcessSecureRandomSetSeed(ProgramState state, IInvocationOperationWrapper invocation)
+    private static ProgramState ProcessSeedingMethods(ProgramState state, IInvocationOperationWrapper invocation)
```

Comment:
This used to track `SecureRandom.SetSeed`, now it ALSO tracks `IRandomGenerator.AddSeedMaterial`